android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/ddm/form/DDMFormView.kt

```diff
@@ -255,7 +255,7 @@ class DDMFormView @JvmOverloads constructor(
 	override fun subscribeToValueChanged(observable: Observable<Field<*>>) {
 		subscription = observable.doOnNext {
 			presenter.fieldModelsChanged(it)
-		}.debounce(2, TimeUnit.SECONDS)
+		}.debounce(500, TimeUnit.MILLISECONDS)
```

Comment:
500ms isn't too little time?

Reply:
For me it's not too little time. If in the demo worked well, we can keep it like that

Reply:
We can make this parameterizable, so the user can choose the best time for him. I'll create a ticket for this.

Reply:
I like that 👍  Thanks!

Reply:
That solution is the most flexible and convenient, yeah. So, just merge this and add that after